demo/store/index.js

```diff
@@ -0,0 +1,19 @@
+import * as mods from "glob:./**/*.slice.js"
+
+const getSlices = connectors =>
+  Object.entries(connectors)
+    .filter(([_, definition]) => definition.hasOwnProperty("slice"))
+    .map(([_, definition]) => [definition.slice.name, definition.slice])
+
+const getThunks = connectors => {
+  let combinedThunks = {}
+  for (let [_, definition] of Object.entries(connectors)) {
+    Object.entries(definition).map(([key, value]) => {
+      if (key !== "slice") combinedThunks[key] = value
+    })
+  }
+  return combinedThunks
+}
```

Comment:
This seems unused, do we need this?

Reply:
It was just something I was exploring, by exporting `export const thunks = getThunks(mods)`, I thought about allowing users to import thunks inside the app easily as:
```js
import { thunks } from "@store"
....
// somewhere in the code
await thunks.article_list()
```
As opposed to importing directly from the connector folder: `import { articles_list } from ../store/articlesAPI/articles.slice` or something like that

It doesn't need to be this way though. I'd like to know your thoughts, I can remove it altogether

Reply:
Ok I removed it, after giving some thought. PR is updated